src/bot/commands/admin/add-category.js

```diff
@@ -16,23 +16,23 @@ export default createCommand({
     // parse matches
     .then(results => parseMatches(results, search))
     // return the matches and output
-    .then(results => prepareAddOutput(results, token))
+    .then(prepareAddOutput)
+        // handle any errors
+    .then(throwIfErrors)
+    // if matches do not exist, add new category
     .then(results => {
-
-      // if matches do not exist, add new category
       const {matches} = results;
       if (matches.length === 0) {
-        return query.categoryInsert({token, name: search})
+        return query.categoryInsert({token, name: search, isActive: true})
           .then(() => {
+            console.log('got it');
```

Comment:
You can get more detailed debugging by starting the dev environment with `DEBUG_SQL=true npm run start-dev`

https://github.com/bocoup/skillsbot/blob/master/src/services/db.js#L12-L15